test/test_write_read.py

```diff
@@ -13,28 +13,29 @@
 
 import os
 import numpy as np
+from netCDF4 import Dataset
 
 from utilities import chdir
-from pyugrid.ugrid import UGrid, UVar
+from pyugrid import UGrid, UVar, load_grid
 from test_examples import two_triangles
 
 
-# find this dir
+# Find this directory.
 test_dir = os.path.split(__file__)[0]
 test_files = os.path.join(test_dir, 'files')
 
 
 def test_with_faces():
     """
-    Test with faces, edges, but no `face_coordintates` or `edge_coordinates`.
+    Test with faces, edges, but no `face_coordinates` or `edge_coordinates`.
 
     """
-
     with chdir(test_files):
         grid = two_triangles()
         grid.save_as_netcdf('2_triangles.nc')
         # Read it back in and check it out.
-        grid2 = UGrid.from_ncfile('2_triangles.nc')
+        with Dataset('2_triangles.nc') as nc:
```

Comment:
Note to self: with statement is safer when reading/saving netCDF files during the tests.